SimpleCam/SimpleCam/SimpleCam.m

```diff
@@ -212,7 +218,7 @@ - (void) setup {
     _capturedImageV.frame = _imageStreamV.frame; // just to even it out
     _capturedImageV.backgroundColor = [UIColor clearColor];
     _capturedImageV.userInteractionEnabled = YES;
-    _capturedImageV.contentMode = UIViewContentModeScaleAspectFill;
+    _capturedImageV.contentMode = UIViewContentModeScaleAspectFit;
```

Comment:
Actually, that's wrong.   I had changed it when I was doing something else.  I'll revert that change.  Thanks for pointing it out.